group.go

```diff
@@ -285,25 +288,26 @@ func (g *Group) fullHeight() int {
 	height := g.selector.Total()
 	g.selector.Range(func(_ int, field Field) bool {
 		height += lipgloss.Height(field.View())
+		height += lipgloss.Height(gap)
```

Comment:
without this line, content that fits in the current window size will truncate its height slightly

what it looks like without this line:
https://github.com/user-attachments/assets/7a3a0d33-2524-49ca-b334-02ba072af10d